include/lldb/Target/Process.h

```diff
@@ -1590,19 +1590,14 @@ class Process : public std::enable_shared_from_this<Process>,
   void PrintWarningOptimization(const SymbolContext &sc);
 
   //------------------------------------------------------------------
-  /// Print a user-visible warning about a module having Swift settings
-  /// incompatible with the current system
-  ///
-  /// Prints a async warning message to the user one time per Process for a
-  /// Module
-  /// whose Swift AST sections couldn't be loaded because they aren't buildable
-  /// on
+  /// Prints a async warning message to the user one time per Process for a Module
+  /// whose Swift AST sections couldn't be loaded because they aren't buildable on
   /// the current machine.
   ///
   /// @param [in] module
   ///     The affected Module.
   //------------------------------------------------------------------
-  void PrintWarningCantLoadSwift(const Module &module);
+  void PrintWarningCantLoadSwift(const Module &module, const char *details);
```

Comment:
Can we rename this function to something more generic? Something like `PrintWarningCantLoadModule`

Reply:
done.